vscode/src/extension.ts

```diff
@@ -51,6 +51,22 @@ function run(exec: string, args: Array<string>): RunResult {
   }
 }
 
+async function missingJagSetupPrompt(jagExec: string) : Promise<boolean> {
+  const setupJagAction = "Setup Jaguar";
+  const action = await Window.showErrorMessage(`The Jaguar installation is incomplete.`, setupJagAction);
+  if (action === setupJagAction) {
+    cp.execFileSync(jagExec, ["setup"]);
+  }
+  const jagResult = run("jag", [ "setup", "--check" ]);
+  if (jagResult.output?.startsWith("Jaguar setup is valid.")) {
```

Comment:
Would be better to use the exit code of the command instead of parsing the output. That assumes that `--check` exits with 1 when the setup isn't correct (which it should).

Reply:
I think it does.

Reply:
I also fixed that it uses `"jag"` instead of the exec path.